gulpfile.js

```diff
@@ -31,16 +35,21 @@ gulp.task('test:browser:before', (done) => {
     swarmB = new Swarm(peerB)
 
     swarmB.transport.add('ws', new WebSockets())
-    swarmB.transport.listen('ws', {}, null, cb)
+    swarmB.transport.listen('ws', {}, null, () => {
+      // swarmB.connection.addStreamMuxer(spdy)
+      // swarmB.connection.reuse()
```

Comment:
@dignifiedquire spdy (or identify) wasn't being used for browser tests and I found that if I run them:

- with spdy - they pass locally, fail on CI
- with identify - they fail

However, this is combination is used in several other places, including orbit. Are these tests missing anything?